client/styles/task-runner.css

```diff
@@ -0,0 +1,73 @@
+/* This file is manually maintained, it is not generated from SCSS sources */
+
+.task {
+  padding-top: 10px;
+}
+
+.task__panel {
+  padding: 10px 15px 15px 15px;
+}
+
+.task__selector {
+    display: inline-block;
+    height: 42px;
+    margin: 0 0 0 15px;
+    visibility: hidden;
+}
+
+.task__selector--universal {
+  width: 105px;
+}
+
+.task__selector--immediate {
+  width: 144px;
+}
+
+.task__selector--queue-only {
+  width: 146px;
+}
+
+.task__selector:checked + .task__label .task__label-inner,
+.task__selector:hover + .task__label .task__label-inner {
+  border-bottom-color: #475266;
```

Comment:
What's this colour for? Should it be #43536d?

Reply:
good catch, thanks :)